components/molecule/dropdownList/src/index.js

```diff
@@ -35,9 +41,14 @@ const MoleculeDropdownList = forwardRef(
     const [typedWord, setTypedWord] = useState('')
     const debouncedTypedWord = useDebounce(typedWord, DEBOUNCE_TIME)
 
-    const classNames = cx(BASE_CLASS, `${BASE_CLASS}--${size}`, {
-      [CLASS_HIDDEN]: !visible
-    })
+    const classNames = cx(
+      BASE_CLASS,
+      `${BASE_CLASS}--${design}`,
```

Comment:
`${BASE_CLASS}--design-${design}` for not matching any other possible value in future